src/components/Tabs/Tabs.tsx

```diff
@@ -65,6 +71,112 @@ const TabsComponent = ({
 
   const withGaps = mode === "accent" || mode === "secondary";
 
+  function getTabEls(): HTMLDivElement[] {
+    if (!tabsRef.current) {
+      return [];
+    }
+
+    return Array.from(
+      // eslint-disable-next-line
+      tabsRef.current.querySelectorAll<HTMLDivElement>(
+        "[role=tab]:not([disabled])"
+      )
+    );
+  }
+
+  function onDocumentKeydown(e: KeyboardEvent) {
```

Comment:
```suggestion
  function handleDocumentKeydown(event: KeyboardEvent) {
```

зачастую используем префикс `handle` для обработчиков и не сокращаем `event`